eng/Version.Details.xml

```diff
@@ -634,22 +634,22 @@
       <SourceBuild RepoName="scenario-tests" ManagedOnly="true" />
     </Dependency>
     <!--
-      Aspire isn't really a toolset dependency. However, it only inserts a baseline manifest in installer,
-      and if you squint at it, this means we can say that its specific dependency versions don't matter to installer.
+      Aspire isn't really a toolset dependency. However, it only inserts a baseline manifest in sdk,
+      and if you squint at it, this means we can say that its specific dependency versions don't matter to sdk.
       It also doesn't currently ship 9.0 preview versions, meaning the version is locked to the latest shipped from 8.0 era.
-      Avoiding this as a product dependency avoids a long coherency path (aspnetcore->extensions->aspire->installer).
+      Avoiding this as a product dependency avoids a long coherency path (aspnetcore->extensions->aspire->sdk).
       **It is** of course possible that an incoherent aspire means that aspire depends on versions of extensions that
       aren't shipping, or those extensions packages depend on aspnetcore packages that won't ship. However, given the cost
       of maintaining this coherency path is high. This being toolset means that aspire is responsible for its own coherency.
     -->
-    <Dependency Name="Microsoft.NET.Sdk.Aspire.Manifest-8.0.100" Version="8.0.1">
+    <Dependency Name="Microsoft.NET.Sdk.Aspire.Manifest-8.0.100" Version="8.1.0">
       <Uri>https://github.com/dotnet/aspire</Uri>
-      <Sha>a6e341ebbf956bbcec0dda304109815fcbae70c9</Sha>
+      <Sha>d304c5f6f15bcd4f34f1841b33870cfab88e6937</Sha>
     </Dependency>
     <!-- Intermediate is necessary for source build. -->
-    <Dependency Name="Microsoft.SourceBuild.Intermediate.aspire" Version="8.0.1-preview.8.24267.1">
+    <Dependency Name="Microsoft.SourceBuild.Intermediate.aspire" Version="8.1.0-preview.1.24373.2">
```

Comment:
hmm this version didn't get published to the dotnet8-transport feed, only to dotnet8-workloads which we don't have in the sdk repo

Reply:
@marcpopMSFT can/should we add the dotnet8-workloads feed?

Reply:
Even if sdk did, note that this was a stable build of Aspire, meaning that while the builds are on the dotnet8-workloads channel, it used a dedicated feed for publishing.

Reply:
The missing package is Microsoft.SourceBuild.Intermediate.aspire which is not a stable version. It looks like the source build intermediate packages get published to the same channel that the original package was published to even if the original was a stable version. I was not aware of this when we were discussing how aspire would publish release versions to the new dotnet8 workloads feeds. Seems like we need to manually add the dotnet8-workloads feed because of how these source build intermediate packages.

Reply:
@marcpopMSFT ok, I added the feed